CodingStandards/ProgramEntryPoint.cs

```diff
@@ -6,32 +6,42 @@
 using System.Threading.Tasks;
 
 class Program
```

Comment:
The class should have XML commentary (see C1). I won't count off this time, but note this for the next assignment.